cypress/support/cypress-commands/assertion.js

```diff
@@ -422,24 +408,26 @@ function loggingValidationCheckResult(validationCheck) {
     }
   });
 
-  // Printing the status of all checks in the report.
-  cy.get(validationCheck)
-    .each((logging) => {
-      cy.log(
-        `${logging.validationPoint}: ${logging.validationStatus}. ${logging.message}`,
-        'loggingValidationCheckResult'
+  // Constructing the logString to pass to fireLog
+  // Assume the checks. If anything is marked other than skipped or pass, then fail the testcase.
+  validationCheck.forEach((assertion) => {
```

Comment:
I foresee a problem here. The reason for the loops is we need logs for all assertions even if one of the assertion fails. I suspect that the use of `firelog` here might break that.

To verify this, make the schema validation fail or run any scenario where schema validation is failing in an existing report. You should see the logs for all 4 validations. If not, changes in this block has to be reverted. 
We can create a new ticket to optimize this flow separately.